tests/cpp/unit-tests/CMakeLists.txt

```diff
@@ -31,110 +31,27 @@ cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_BUILD_TYPE DEBUG)
 set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_FLAGS --coverage)
+if((CMAKE_CXX_COMPILER_ID STREQUAL "GNU") AND (CMAKE_C_COMPILER_ID STREQUAL "GNU"))
+    set(CMAKE_CXX_FLAGS --coverage)
+    add_link_options(--coverage)
+endif()
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
-find_library(SR_LIB smartredis PATHS ../../../install/lib NO_DEFAULT_PATH REQUIRED)
+find_library(REDISPP redis++    PATHS ../../../install/lib NO_DEFAULT_PATH REQUIRED)
+find_library(HIREDIS hiredis    PATHS ../../../install/lib NO_DEFAULT_PATH REQUIRED)
+find_library(SRLIB   smartredis PATHS ../../../install/lib NO_DEFAULT_PATH REQUIRED)
+find_library(SRLIB_FORTRAN   smartredis-fortran PATHS ../../../install/lib NO_DEFAULT_PATH REQUIRED)
 find_package(Threads REQUIRED)
+set(EXT_LIBRARIES ${REDISPP} ${HIREDIS} ${SRLIB} ${SRLIB_FORTRAN})
 
 include_directories(SYSTEM
     /usr/local/include
     ../../../include
 	../../../install/include
```

Comment:
Not on you, but the spacing of this line is off

Reply:
Good eye. This has been fixed